src/main/kotlin/step2/ArithmeticOperation.kt

```diff
@@ -0,0 +1,38 @@
+package step2
+
+enum class ArithmeticOperation(
+    private val expression: Char
+) {
+    PLUS('+') {
+        override fun execute(firstNum: Int, secondNum: Int): Int {
+            return firstNum + secondNum
+        }
+    },
```

Comment:
enum 클래스의 멤버 변수로 람다를 갖게 한다면 더 깔끔하게 작성하실 수 있습니다 🙂

```suggestion
enum class ArithmeticOperation(
    private val expression: Char,
    val calculate: (Int, Int) -> Int
) {
    PLUS('+', { firstNum, secondNum -> firstNum + secondNum }),
```